laborator/content/rolul-registrelor-adresare/1-2-multiply/multiply.asm

```diff
@@ -24,10 +24,7 @@ main:
     mul bl
 
     ; Print result in hexa
-    PRINTF32 `%s\x0`, print_mesaj
-    xor ebx, ebx
-    mov bx, ax
-    PRINTF32 `%hx\n\x0`, eax
+    PRINTF32 `%s%hx\n\x0`, print_mesaj, eax
```

Comment:
```suggestion
    PRINTF32 `Rezultatul este 0x%hx\n\x0`, print_mesaj, eax
```
`print_mesaj` becomes kind of useless at this point. It was only used during ancient times because the SASM macros couldn't take string literals.